src/main/java/Duke.java

```diff
@@ -1,10 +1,125 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 public class Duke {
+    static final Pattern DEADLINE_PATTERN = Pattern.compile("(.+)/by (.+)");
+    static final Pattern EVENT_PATTERN = Pattern.compile("(.+)/from (.+) /to (.+)");
+
+    public static ArrayList<Task> tasks = new ArrayList<>();
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printASCII();
+
+        try (Scanner sc = new Scanner(System.in)) {
+            String command;
+            do {
+                command = sc.next();
+                String body = sc.nextLine();
+
+                switch (command) {
+                    case ("bye"):
```

Comment:
one suggestion would be to make commands enums, for example:  enum CommandType {BYE, LIST, UNMARK...} etc